packages/docs/src/components/api/ApiSection.vue

```diff
@@ -76,65 +30,23 @@
     name: 'ApiSection',
 
     props: {
-      hideHeader: Boolean,
       name: String,
-      page: String,
       section: String,
     },
 
     data: () => ({
-      apiComponents: [],
-      selectedComponent: {},
       filter: null,
+      apiData: [],
     }),
 
-    computed: {
-      inlineApi: get('user/api'),
-      toc: sync('pages/toc'),
-      apiComponent () {
-        return Object.keys(this.selectedComponent)
-          .filter(key => this.section ? this.section === key : !['component', 'mixins', 'name'].includes(key))
-          .reduce((obj, key) => {
-            if (this.selectedComponent[key].length) {
-              obj[key] = this.selectedComponent[key]
-            }
-            return obj
-          }, {})
-      },
-    },
-
     async created () {
-      const components = this.name ? [this.name] : pageToApi[this.page] || []
-
-      if (!components.length) {
-        throw new Error(`API for ${this.name || this.page} does not exist`)
-      }
+      const api = (await getApi(this.name)).default
```

Comment:
could this be in a try catch that errors if it doesn't exist, or if the async call fails?